src/uct/ib/base/ib_iface.h

```diff
@@ -89,9 +91,26 @@ typedef struct uct_ib_address_pack_params {
     /* GID index,
        must be valid if @ref UCT_IB_ADDRESS_PACK_FLAG_GID_INDEX is set. */
     uint8_t                           gid_index;
+    /* PKEY value,
+       must be valid if @ref UCT_IB_ADDRESS_PACK_FLAG_PKEY is set. */
+    uint16_t                          pkey;
 } uct_ib_address_pack_params_t;
 
 
+typedef struct uct_ib_unpack_address_params {
```

Comment:
can we unite uct_ib_address_pack_params and uct_ib_unpack_address_t to same struct?